### PR TITLE
Updated Diagnostic Report with two new columns for cdwId

### PR DESCRIPTION
**Pull request by taylorrassmannlibertyits**
1. CDWIdNumber
2. CDWIdResourceCode

### CODE REVIEW

**Review by nmoisan — COMMENTED** [review]
LGTM. If I remember correctly, the CDWId column is removed later?

**Comment by taylorrassmannlibertyits** [comment]
> LGTM. If I remember correctly, the CDWId column is removed later?

Yes I think so.

File: db/migration/V18_0__drop_and_create_diagnostic_report.sql [hunk 1]
```diff
@@ -0,0 +1,29 @@
+drop table [App].[DiagnosticReport_V2]
+go
+
+Create table [App].[DiagnosticReport_V2]
+(
+  [CDWId] varchar(15) not null,
```
Comment (nmoisan) [inline db/migration/V18_0__drop_and_create_diagnostic_report.sql:6]: BIGINT makes this a `varchar(26)` on our end. For consistency should this also be `varchar(26)`?

Comment (taylorrassmannlibertyits) [reply db/migration/V18_0__drop_and_create_diagnostic_report.sql:6]: Sure I will make the update

**Review by nmoisan — APPROVED** [review]

**Review by bschofield-va — APPROVED** [review]

**Pull request merged**